components/Profile/index.js

```diff
@@ -77,16 +76,8 @@ function Profile({ id, type }) {
                 {type}
               </Title>
             </Col>
-            {info.belt ? (
-              <Col span={24}>
```

Comment:
The `<Col span={24}>` should be here, and not extracted to the component. Only <Tag> :bulb: